opendevin/core/config.py

```diff
@@ -490,6 +492,13 @@ def get_parser():
         type=int,
         help='The maximum number of iterations to run the agent',
     )
+    parser.add_argument(
+        '-b',
+        '--max-budget-per-task',
+        default=config.max_budget_per_task,
+        type=float,
+        help='The maximum budget allowed per task, beyond which the agent will stop.',
+    )
```

Comment:
I think eventually we may want to get rid of these CMDLINE arguments otherwise our backend-only mode`main.py` would diverge more from fullstack-mode (which doesn't use cmdline args).

Don't get me wrong, this is totally fine here.